var/da/da_main/da_solve.inc

```diff
@@ -1129,6 +1129,9 @@
        (config_flags%real_data_init_type == 3)) then
       if ( .not. offline_varbc ) &
       call da_update_firstguess(input_grid)
+!mchen@ium.cn---
+      call da_write_anaincrements (grid, config_flags)
+!mchen@ium.cn---
```

Comment:
Should you delete two commented-out lines with mchen?

Reply:
I think we should make this IAU writing optional.

Reply:
@liujake The analysis increment file is only written out if several namliests associated with auxhist5 are added in the namelist.input file, for example, auxhist5_interval, io_form_auxhist5.

Reply:
Ok. Looks like controlling optional writing is inside the write subroutine.